docs/install_guides/installing-openvino-linux.md

```diff
@@ -1,52 +1,40 @@
 # Install and Configure Intel® Distribution of OpenVINO™ toolkit for Linux* {#openvino_docs_install_guides_installing_openvino_linux}
```

Comment:
Let's remove asterisks for all third-party product names (it was approved by Marketing): Windows, Linux, Ubuntu etc